daemon/graphdriver/driver_windows.go

```diff
@@ -0,0 +1,26 @@
+package graphdriver
+
+type DiffDiskDriver interface {
+	Driver
+	CopyDiff(id, sourceId string) error
+}
+
+const (
+	FsMagicWindows = FsMagic(0xa1b1830f)
+)
+
+var (
+	// Slice of drivers that should be used in an order
+	priority = []string{
+		"windows",
+	}
+
+	FsNames = map[FsMagic]string{
+		FsMagicWindows:     "windows",
```

Comment:
The Windows graphdriver will effectively be passthrough and the implementation far lower down the stack in win32 - if there is an implementation change, it will be handled down at that layer. There are no plans to offer multiple drivers.